<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindPackage</class>
 <widget class="QDialog" name="FindPackage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>2000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find Package</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>800</width>
     <height>400</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="txt">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555;
font: 6pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
        </property>
        <property name="text">
         <string>Package Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPackageName">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFindPackage">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #555;
font: 6pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
        </property>
        <property name="text">
         <string>Find Package</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
